packages/core/src/utils/ans/utils/ans-name-to-ibc-token-name.ts

```diff
@@ -0,0 +1,6 @@
+import { CHAIN_SEPARATOR } from '../constants'
+export function getIbcTokenName(chainName: string, asset: string) {
```

Comment:
```suggestion
export function getIbcTokenAnsName(chainName: string, asset: string) {
```

Reply:
This is also a question - do we want to clarify that "name" is ANS specific or is "name" of an asset specific enough?

Reply:
Was renamed to `ibcTokenToAnsId`